src/lib/y2storage/boot_requirements_strategies/base.rb

```diff
@@ -22,61 +22,61 @@
 # find current contact information at www.suse.com.
 
 require "yast"
-require "storage/patches"
 require "y2storage/disk_size"
 require "y2storage/filesystems/type"
 require "y2storage/planned"
+require "y2storage/boot_requirements_strategies/analyzer"
+require "y2storage/exceptions"
 
 module Y2Storage
   module BootRequirementsStrategies
-    class Error < RuntimeError
+    class Error < Y2Storage::Error
```

Comment:
For example, in `Y2Storage::Proposal::AutoinstProposal` and even on `yast2-autoinstallation` (the next PR).